recipes/openmc-plotter/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "openmc-plotter" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openmc-plotter-{{ version }}.tar.gz
+  sha256: 06e9e17f12f119e81b9fbf80cee9276cf3ec144a6be9f783b304646d357c32b4
+build:
+  entry_points:
+    - openmc-plotter=openmc_plotter.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
```

```suggestion
    - python  >=3.6
```
I think GitHub Actions and therefore the linter are down.